### ISSUE TITLE
iOS support

### ISSUE THREAD

**Issue by dokar3** [dokar3/ChipTextField#131]
Not ready yet, but as always, it's **open to contribute!**

**Comment by kagg886** [comment]
hey！ I plan to support the IOS platform, and I have a macOS machine(borrowed from my friend). 

if this project worked on my machine, i want to send a PR. Please merge my code as soon as possible and release the latest version to support IOS thanks! :D

**Comment by kagg886** [comment]
@N7ghtm4r3 focus on #172

**Comment by dokar3** [comment]
@N7ghtm4r3 Yep, it's still missing. Haven't been actively working on Kotlin these days.

**Comment by N7ghtm4r3** [comment]
> > Hi, `iOs` still missing?
> 
> Yes. but i clone it and support iOS

Hi @magic-cucumber thank you!

**Comment by N7ghtm4r3** [comment]
> @N7ghtm4r3 Yep, it's still missing. Haven't been actively working on Kotlin these days.

Hi @dokar3 ! I hope that can be merged the porting by @magic-cucumber

**Comment by dokar3** [comment]
@N7ghtm4r3 A new alpha version is out, iOS targets included. Might or might not work, feel free to try.
https://github.com/dokar3/ChipTextField/releases/tag/v0.7.2-alpha01

**Comment by N7ghtm4r3** [comment]
Thank you!

**Comment by magic-cucumber** [comment]
> @N7ghtm4r3 A new alpha version is out, iOS targets included. Might or might not work, feel free to try.
> https://github.com/dokar3/ChipTextField/releases/tag/v0.7.2-alpha01

oh yes！
